internal/log/logger.go

```diff
@@ -0,0 +1,7 @@
+package log
+
+type Logger interface {
+	Error(err error)
+	Warn(msg string)
+	Info(msg string)
+}
```

Comment:
Would logger interface not be better implemented as: ?
```go
type Logger interface {
	Error(msg string)
	ErrorV(msg string, v interface{})
	Warn(msg string)
	WarningV(msg string, v interface{})
	Info(msg string)
	InfoV(msg string, v interface{})
}
```

might also be nice with `XxxxxF(format string, ...v interface{})`